src/catalog/catalog_set.cpp

```diff
@@ -50,12 +50,24 @@ CatalogEntry* CatalogSet::getEntryNoLock(Transaction* transaction, const std::st
     return entry;
 }
 
+static void LogEntryForTrx(Transaction* transaction, CatalogSet& set, CatalogEntry& entry) {
```

Comment:
LogEntryForTrx -> logEntryForTrx.

Function names always stay as the first letter lower case regardless of static or non-static